Documentation/docs/learn/dicom.md

```diff
@@ -0,0 +1,118 @@
+# DICOM
+
+## DICOM: Rescale Slope/Intercept
+
+Another question that is often asked on insight-users is: I am trying
```

Comment:
We can remove *on insight-users* because I think it will just confuse most folks who will not be familiar with the history of the mailing lists.

Reply:
:+1: Done in c4c66f3.